models/migrations/v1_20/v255.go

```diff
@@ -12,18 +12,18 @@ import (
 func CreateActionArtifactTable(x *xorm.Engine) error {
 	type ActionArtifact struct {
 		ID               int64 `xorm:"pk autoincr"`
-		JobID            int64 `xorm:"index"`
+		RunID            int64 `xorm:"index unique(run-id-name)"` // The run id of the artifact
 		RunnerID         int64
 		RepoID           int64 `xorm:"index"`
 		OwnerID          int64
 		CommitSHA        string
-		StoragePath      string // The path to the artifact in the storage
-		FileSize         int64
-		FileGzipSize     int64
-		ContentEncnoding string             // The content encoding of the artifact, such as gzip
+		StoragePath      string             // The path to the artifact in the storage
+		FileSize         int64              // The size of the artifact in bytes
+		FileGzipSize     int64              // The size of the artifact in bytes after gzip compression
+		ContentEncnoding string             // The content encoding of the artifact
 		ArtifactPath     string             // The path to the artifact when runner uploads it
-		ArtifactName     string             // The name of the artifact when runner uploads it
-		UploadStatus     int64              `xorm:"index"`
+		ArtifactName     string             `xorm:"unique(run-id-name)"` // The name of the artifact when runner uploads it
```

Comment:
Does database like such `a-b-c` index name? IIRC we always underline `_` before?